code/modules/urist/modules/GOAI/integrations/base_commander/mob_commander/obstacle_handlers/generic.dm

```diff
@@ -0,0 +1,60 @@
+/datum/goai/mob_commander/proc/HandlePry(var/datum/ActionTracker/tracker, var/obj/target)
+	log_debug("Pry task for [target]")
+	if(src.brain.GetMemoryValue(MEM_OBSTRUCTION, null) == target)
+		src.brain.DropMemory(MEM_OBSTRUCTION)
+	tracker.SetDone()
+	return
+	//TODO
+
+/datum/goai/mob_commander/proc/HandleScrew(var/datum/ActionTracker/tracker, var/obj/target)
+	log_debug("Screw task for [target]")
+	tracker.SetDone()
+	return
+	//TODO
+
+/datum/goai/mob_commander/proc/HandleGenericBreak(var/datum/ActionTracker/tracker, var/obj/target, var/method)
+	log_debug("Break task for [target]")
+	var/mob/pawn = src.GetPawn()
```

Comment:
Can we rename this to mob_pawn? I'd like to keep plain 'pawn' to be an atom by convention to make it explicit what logic is supported for all objects vs just mobs vs just humanoids.

Reply:
Absolutely!